starlette/testclient.py

```diff
@@ -58,11 +65,11 @@ def send(self, request, *args, **kwargs):
 
         # Include the 'host' header.
         if "host" in request.headers:
-            headers = []
+            headers = []  # type: list
         elif port == 80:
             headers = [[b"host", host.encode()]]
         else:
-            headers = [[b"host", ("%s:%d" % (host, port)).encode()]]
+            headers = [[b"host", ("%s:%d" % (host, port)).encode()]]  # type: ignore
```

Comment:
How come we need `type: ignore` here? Also should it be a list of two-tuples instead?